docs/src/global/componentApiTable.jsx

```diff
@@ -1,48 +1,177 @@
 import {
+    capitalize,
+    isString,
+    lowerCase,
     map,
 } from 'lodash';
-import { Table } from 'react-cm-ui';
+import {
+    Table,
+    Tooltip,
+    Typography,
+} from 'react-cm-ui';
+import makeStyles from 'react-cm-ui/styles/makeStyles';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { DOCS_PROPS_PROP_TYPE } from './componentApiConstants';
+import MarkdownContainer from './markdownContainer';
+import { REPLACE_BACKTICKS_WITH_TAG } from './utils';
```

Comment:
Why is this function name in UPPER_SNAKE_CASE?

Reply:
Fixed.

Reply:
Yeah, looks better pascalCased.  Thanks.  Resolving.